### PR TITLE
Improve cheatsheet, add author

### LINKED ISSUE #4: Cheatsheet does not show asciidoc syntax

### PR DESCRIPTION
**Pull request by madmas**
Change cheatsheet to reflect asciidoc syntax
fixes  #4

### CODE REVIEW

**Review by PReimers — APPROVED** [review]
looks good for me 😉

File: AUTHORS [hunk 1]
```diff
@@ -2,11 +2,13 @@ List of HackADoc contributors.
 
 Hackergarten Basel
 
+fliiiix 
 ursjoss
 PReimers
 madmas
 
 
+
```
Comment (PReimers) [inline AUTHORS:11]: @madmas 
Why did you add this additional blank line?

**Review by PReimers — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (madmas) [reply AUTHORS:11]: to have more of them ;-)
